src/testing/docker/docker-tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+./default-docker-test.sh
+./docker-compose-test.sh
```

Comment:
The tests should run independently. This would allow us to see
- which test fails
- whether docker compose is able to build everything from scratch